tt_metal/hw/ckernels/blackhole/metal/llk_api/llk_sfpu/ckernel_sfpu_dropout.h

```diff
@@ -1,60 +1,24 @@
-// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+// SPDX-FileCopyrightText: © 2025 Tenstorrent Inc.
```

Comment:
```suggestion
// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
```
Year denotes the year of file creation, not the time of modification.